content/posts/0.24-release.md

```diff
@@ -90,6 +90,8 @@ was totally silent and it was hard to tell which versions of which packages were
 being installed from which indices. Now, when run in verbose mode, pip-like
 installation messages are printed to stdout.
 
+For a detailed list of changes, see the [v0.24 changelog](https://pyodide.org/en/0.24.0/project/changelog.html#version-0-24-0).
+
```

Comment:
Maybe put this at the very top just before `##  Performance Improvements`?

Reply:
Or here is pretty reasonable too.

Reply:
Yeah, I thought about it too but didn't want to direct users to a different page at the top of the blog post. In the end seemed better.